code/modules/organs/organ_internal.dm

```diff
@@ -130,7 +130,25 @@
 
 	if(owner.life_tick % PROCESS_ACCURACY == 0)
 
-		//High toxins levels are dangerous
+		//A liver's duty is to get rid of toxins
+		if(owner.getToxLoss() > 0 && owner.getToxLoss() <= 3)
+			owner.adjustToxLoss(-0.2) //there isn't a lot of toxin damage, so we're going to be chill and slowly filter it out
+		else if(owner.getToxLoss() > 3)
+			if(is_bruised())
+				//damaged liver works less efficiently
+				owner.adjustToxLoss(-0.5)
```

Comment:
removing 0.5 instead of 0.2 seems more efficient to me?

Reply:
A healthy liver removes 1 tox per one of these if there's more than 3 toxins in the body. A bruised liver removes 0.5 per one of these.